Test/Load/setup-files/LoadTestCartData.php

```diff
@@ -0,0 +1,217 @@
+<?php
```

Comment:
At a high level this seems to only load test this endpoint? What are we hoping that load testing this test endpoint which is not used anywhere will give us? I might be missing some context.

Wouldn't a better load test be to extract some of the endpoints we actually have and load test them?

Reply:
You are right, and we will be testing the other endpoints of the plugin, but I split the actual tests into a separate PR found here: https://github.com/BoltApp/bolt-magento2/pull/429

This PR is just to expose this part of the plugin that is triggered each time we get to the cart page (i.e `getBoltpayOrder()` which converts the cart on a merchant site to a bolt order ). 

But, for more context the load tests found in the other PR will test the user flow through checkout: 
- Getting to the cart page (which creates the order token and order reference)
- Calling the shipping and tax web hook set up on the merchant's site
- Calling the Pre-auth web hook set up on the merchant's site 

The purpose behind these load tests is to see whether our plugin is the actual problem when a merchant has a significant number of users accessing their store.